src/open_inwoner/configurations/bootstrap/siteconfig.py

```diff
@@ -0,0 +1,105 @@
+from django.conf import settings
+
+from django_setup_configuration.configuration import BaseConfigurationStep
+
+from open_inwoner.configurations.models import SiteConfiguration
+
+
+class SiteConfigurationStep(BaseConfigurationStep):
+    """
+    Set up general configuration ("Algemene configuratie")
+    """
+
+    verbose_name = "Site configuration"
+    required_settings = [
+        "SITE_NAME",
+        "SITE_SECONDARY_COLOR",
+        "SITE_ACCENT_COLOR",
```

Comment:
Is there a reason why the primary_color isn't required?

Reply:
The `SiteConfiguration` model has a default for the primary color, so I thought it's not necessary to make this required?

Reply:
Hmm that's true, though I do feel like if a municipality is automating this, the primary color of the website (and other colors in general) is something they should definitely always set explicitly

Reply:
Makes sense, I'll add it.